From: Chris P <sahib@online.de>
Date: Mon, 16 Jan 2017 11:35:51 +0100
Subject: Cherry-pick unreleased upstream commit.

Description: Cherry-pick unreleased upstream commit.
 utilites.c: Remove forgotten debug print
Author: Christopher Pahl <sahib@online.de>
Origin: upstream, https://github.com/sahib/rmlint/commit/623d34811474776e2a0e1d67544f9cb3f75e7c78
Last-Update: 2017-01-16
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
---
 lib/utilities.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/lib/utilities.c b/lib/utilities.c
index 163547e..a53639f 100644
--- a/lib/utilities.c
+++ b/lib/utilities.c
@@ -871,7 +871,6 @@ dev_t rm_mounts_get_disk_id(RmMountTable *self, _UNUSED dev_t dev, _UNUSED const
     char *prev = g_strdup(path);
     while(TRUE) {
         char *parent_path = g_path_get_dirname(prev);
-        g_printerr("PREV %s %s\n", prev, parent_path);
 
         RmStat stat_buf;
         if(!rm_sys_stat(parent_path, &stat_buf)) {
